admin_guide/master_node_configuration.adoc

```diff
@@ -10,44 +10,116 @@
 toc::[]
 
 == Overview
-`openshift start`, `openshift start master`, and `openshift start node` all take a limited set of arguments that are sufficient for development and experimental purposes, but are insufficient to describe and control the full set of configuration and security options that are necessary in a production environment.  To provide those options, it is necessary to use the dedicated master and node config files.
+The `openshift start` command is used to launch OpenShift servers. The command
+and its subcommands, `master` to launch a
+link:../architecture/infrastructure_components/kubernetes_infrastructure.html#master[master
+server] and `node` to launch a
+link:../architecture/infrastructure_components/kubernetes_infrastructure.html#node[node
+server], all take a limited set of arguments that are sufficient for launching
+servers in a development or experimental environment.
 
-The config files are fully specifying with no defaulting.  This means that any empty value means that you want to start up with an empty value for that parameter.  It makes it easy to reason about exactly what your configuration is, but it also means that it can be difficult to remember all of the options to specify.  To make this easier, the config files can be created with the `--write-config` flag and can be used via the `--config` flag.
+However, these arguments are insufficient to describe and control the full set
+of configuration and security options that are necessary in a production
+environment. To provide those options, it is necessary to use the dedicated
+master and node configuration files.
 
-== Create the starting config files
-The `openshift start` command accepts flags that indicate that it should simply write the config file that it would have used and terminate.  This is useful for getting a starting point for the config.  You can do this by running 
+The configuration files for masters and nodes are fully specifying with no
+defaulting. This means that any empty value means that you want to start up with
```

Comment:
"This means that any empty value means..."
Unclear. Having two "means" this close could be better expressed as: "Therefore, any empty value indicates that you want to start up with an empty value for that parameter."